docs/style-guide.md

```diff
@@ -44,8 +44,8 @@ const fs = Promise.promisifyAll(require('fs-extra'))
 
 ### Leave a blank line (EOL) at the end of file
 
-### Comments
-Feel free to leave comments where necessary.
+### Code Comments
+Feel free to comment your code when necessary.
```

Comment:
commenting code sometimes means changing codes into comments, it is a little ambiguous